ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLandOwnership.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <v-card flat rounded id="mhr-home-land-ownership" class=" mhr-home-land-ownership">
+    <v-row no-gutters>
+      <v-col cols="12" sm="2">
+        <label class="generic-label" for="ownership">
+        Land Lease or Ownership
+        </label>
+      </v-col>
+      <v-col cols="12" sm="10">
+        <v-checkbox
+          id="ownership"
+          :label="landOwnershipLabel"
+          v-model="isOwnLand"
+          class="my-0 py-0 px-0 ownership-checkbox"
+          data-test-id="ownership-checkbox"
+        />
+       </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
+import { useGetters, useActions } from 'vuex-composition-helpers'
+
+export default defineComponent({
+  name: 'HomeLandOwnership',
+  setup () {
+    const {
+      getMhrRegistrationOwnLand
+    } = useGetters<any>([
+      'getMhrRegistrationOwnLand'
+    ])
+
+    const {
+      setMhrRegistrationOwnLand
+    } = useActions<any>([
+      'setMhrRegistrationOwnLand'
+    ])
+
+    const localState = reactive({
+      isOwnLand: Boolean(getMhrRegistrationOwnLand.value || false),
+      landOwnershipLabel: `The manufactured home is located on land that the homeowners own,
```

Comment:
Doesn't need to be reactive, there is nothing dynamic about this string. You can add directly into the template.

Reply:
Thanks will do.